src/dev_utils/peer.rs

```diff
@@ -40,10 +33,10 @@ pub struct PeerId {
 
 impl PeerId {
     pub fn new(id: &str) -> Self {
-        PEERS
+        NAMES
             .iter()
+            .map(|name| PeerId::new_with_keypair(name))
```

Comment:
What's the point of this lookup? Can't we just do the same thing we do in `new_with_keypair` (and remove `new_with_keypair`)?